client/src/main/java/io/split/client/impressions/strategy/ProcessImpressionStrategy.java

```diff
@@ -0,0 +1,11 @@
+package io.split.client.impressions.strategy;
+
+import io.split.client.impressions.Impression;
+import io.split.client.impressions.ImpressionsResult;
+
+import java.util.List;
+
+public interface ProcessImpressionStrategy {
+
+    ImpressionsResult processImpressions(List<Impression> impressions);
```

Comment:
maybe the name can be just `process` ??? because the interface name is already `processImpression...`